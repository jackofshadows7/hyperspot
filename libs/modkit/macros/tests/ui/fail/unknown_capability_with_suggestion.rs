use modkit_macros::module;

#[module(name="x", capabilities=[rst])]
pub struct X;

fn main() {}

