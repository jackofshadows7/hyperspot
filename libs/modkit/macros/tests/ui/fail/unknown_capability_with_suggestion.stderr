error: unknown capability 'rst'
              = help: did you mean one of: rest, rest_host?
 --> tests/ui/fail/unknown_capability_with_suggestion.rs:3:34
  |
3 | #[module(name="x", capabilities=[rst])]
  |                                  ^^^
